spec/requests/deactivate_user_on_failed_charge_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.